Applying permission update to discord_role_everyone causes plugin crash

I think a fix similar to https://github.com/Lucky3028/terraform-provider-discord/commit/e45dcbb8a244e2adcc8238393aadb4a8848eafdd needs to be applied to https://github.com/Lucky3028/terraform-provider-discord/blob/master/discord/resource_discord_role_everyone.go as well?

Logs:
```
Terraform will perform the following actions:

  # discord_role_everyone.everyone will be updated in-place
  ~ resource "discord_role_everyone" "everyone" {
        id          = "1031055347319832666"
      ~ permissions = 67174401 -> 139653873665
        # (1 unchanged attribute hidden)
    }

Plan: 0 to add, 1 to change, 0 to destroy.

Do you want to perform these actions?
  Terraform will perform the actions described above.
  Only 'yes' will be accepted to approve.

  Enter a value: yes

discord_role_everyone.everyone: Modifying... [id=1031055347319832666]
╷
│ Error: Request cancelled
│ 
│   with discord_text_channel.welcome,
│   on main.tf line 35, in resource "discord_text_channel" "welcome":
│   35: resource discord_text_channel welcome {
│ 
│ The plugin.(*GRPCProvider).PlanResourceChange request was cancelled.
╵
╷
│ Error: Request cancelled
│ 
│   with discord_text_channel.character_creation,
│   on main.tf line 49, in resource "discord_text_channel" "character_creation":
│   49: resource discord_text_channel character_creation {
│ 
│ The plugin.(*GRPCProvider).UpgradeResourceState request was cancelled.
╵
╷
│ Error: Request cancelled
│ 
│   with discord_text_channel.general,
│   on main.tf line 64, in resource "discord_text_channel" "general":
│   64: resource discord_text_channel general {
│ 
│ The plugin.(*GRPCProvider).UpgradeResourceState request was cancelled.
╵
╷
│ Error: Request cancelled
│ 
│   with discord_text_channel.bot_test,
│   on main.tf line 71, in resource "discord_text_channel" "bot_test":
│   71: resource discord_text_channel bot_test {
│ 
│ The plugin.(*GRPCProvider).UpgradeResourceState request was cancelled.
╵
╷
│ Error: Request cancelled
│ 
│   with discord_role_everyone.everyone,
│   on roles.tf line 1, in resource "discord_role_everyone" "everyone":
│    1: resource discord_role_everyone everyone {
│ 
│ The plugin.(*GRPCProvider).ApplyResourceChange request was cancelled.
╵

Stack trace from the terraform-provider-discord_v1.1.0 plugin:

panic: interface conversion: interface {} is int, not *disgord.PermissionBit

goroutine 80 [running]:
github.com/lucky3028/discord-terraform/discord.resourceRoleEveryoneUpdate({0x1b3a6a0?, 0xc000212e40?}, 0xc000487800, {0x1863620?, 0xc000034480?})
        github.com/lucky3028/discord-terraform/discord/resource_discord_role_everyone.go:76 +0x3d9
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).update(0xc0002b10a0, {0x1b3a6a0, 0xc000212e40}, 0xd?, {0x1863620, 0xc000034480})
        github.com/hashicorp/terraform-plugin-sdk/v2@v2.23.0/helper/schema/resource.go:741 +0x12e
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).Apply(0xc0002b10a0, {0x1b3a6a0, 0xc000212e40}, 0xc000286b60, 0xc000487680, {0x1863620, 0xc000034480})
        github.com/hashicorp/terraform-plugin-sdk/v2@v2.23.0/helper/schema/resource.go:847 +0x83a
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*GRPCProviderServer).ApplyResourceChange(0xc000439ed8, {0x1b3a6a0?, 0xc000212d20?}, 0xc00078c370)
        github.com/hashicorp/terraform-plugin-sdk/v2@v2.23.0/helper/schema/grpc_provider.go:1021 +0xe8d
github.com/hashicorp/terraform-plugin-go/tfprotov5/tf5server.(*server).ApplyResourceChange(0xc0003295e0, {0x1b3a6a0?, 0xc000212510?}, 0xc0001c8e70)
        github.com/hashicorp/terraform-plugin-go@v0.14.0/tfprotov5/tf5server/server.go:818 +0x574
github.com/hashicorp/terraform-plugin-go/tfprotov5/internal/tfplugin5._Provider_ApplyResourceChange_Handler({0x19b2b80?, 0xc0003295e0}, {0x1b3a6a0, 0xc000212510}, 0xc0001c8e00, 0x0)
        github.com/hashicorp/terraform-plugin-go@v0.14.0/tfprotov5/internal/tfplugin5/tfplugin5_grpc.pb.go:385 +0x170
google.golang.org/grpc.(*Server).processUnaryRPC(0xc0002461e0, {0x1b3e7c0, 0xc000007ba0}, 0xc0002c70e0, 0xc000452900, 0x20add60, 0x0)
        google.golang.org/grpc@v1.48.0/server.go:1295 +0xb2b
google.golang.org/grpc.(*Server).handleStream(0xc0002461e0, {0x1b3e7c0, 0xc000007ba0}, 0xc0002c70e0, 0x0)
        google.golang.org/grpc@v1.48.0/server.go:1636 +0xa2f
google.golang.org/grpc.(*Server).serveStreams.func1.2()
        google.golang.org/grpc@v1.48.0/server.go:932 +0x98
created by google.golang.org/grpc.(*Server).serveStreams.func1
        google.golang.org/grpc@v1.48.0/server.go:930 +0x28a

Error: The terraform-provider-discord_v1.1.0 plugin crashed!
```

--- a/discord/resource_discord_role_everyone.go
+++ b/discord/resource_discord_role_everyone.go
@@ -72,8 +72,9 @@ func resourceRoleEveryoneUpdate(ctx context.Context, d *schema.ResourceData, m i
 
 	serverId := getId(d.Get("server_id").(string))
 	d.SetId(serverId.String())
+	newPermission := disgord.PermissionBit(d.Get("permissions").(int))
 	if role, err := client.Guild(serverId).Role(serverId).Update(&disgord.UpdateRole{
-		Permissions: d.Get("permissions").(*disgord.PermissionBit),
+		Permissions: &newPermission,
 	}); err != nil {
 		return diag.Errorf("Failed to update role %s: %s", d.Id(), err.Error())
 	} else {

